### PR TITLE
Add tests status modal

### LINKED ISSUE #14: [Page: Editor] {Safari} Test results look slightly "off"
Screen Shot 2021-05-17 at 8 33 09 AM

Not sure what this is supposed to look like, but it's a bit off looking now.

Per Jeremy:

> Looks like summary elements don't style well in safari!

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-project-management/issues/243.
Fixes https://github.com/exercism/v3-beta/issues/14

### CODE REVIEW

File: app/javascript/components/student/iteration-page/TestsInformation.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { TestRunSummary } from '../../editor/TestRunSummary'
+import { TestRun } from '../../editor/types'
+
+const DEFAULT_ERROR = new Error('Unable to fetch test run')
+
+type APIResponse = {
+  testRun: TestRun
+}
+
+export const TestsInformation = ({
```
Comment (kntsoriano) [inline app/javascript/components/student/iteration-page/TestsInformation.tsx:15]: This looks similar to `<TestsFailedModal />` but I decided not to DRY them up yet.

**Comment by iHiD** [comment]
@kntsoriano There's a great bug here for you to track down: https://www.loom.com/share/23deeef9722244edaa83a1f805eddef7

File: app/javascript/components/editor/TestRunSummaryContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import { TestRun, TestRunStatus } from './types'
+import { TestRunChannel } from '../../channels/testRunChannel'
+import { fetchJSON } from '../../utils/fetch-json'
+import { useRequestQuery } from '../../hooks/request-query'
+import { useIsMounted } from 'use-is-mounted'
+import { TestRunSummary } from './TestRunSummary'
+
+const REFETCH_INTERVAL = 2000
+
+export const TestRunSummaryContainer = ({
```
Comment (ErikSchierboom) [inline app/javascript/components/editor/TestRunSummaryContainer.tsx:11]: I like how the callbacks are now no longer in the same file as most of the markup 👍

File: app/javascript/components/student/iteration-page/TestsInformation.tsx (hunk 1 above)
Comment (ErikSchierboom) [reply app/javascript/components/student/iteration-page/TestsInformation.tsx:15]: Yeah no need to do that I think.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**